src/js/components/NavTabsComponent.jsx

```diff
@@ -10,12 +14,32 @@ var NavTabsComponent = React.createClass({
     tabs: React.PropTypes.array.isRequired
   },
 
+  getInitialState: function () {
+    return {
+      activeDeployments: DeploymentStore.deployments.length
+    };
+  },
+
   getDefaultProps: function () {
     return {
       className: ""
     };
   },
 
+  componentWillMount: function () {
+    DeploymentStore.on(DeploymentEvents.CHANGE, () => {
+      this.setState({activeDeployments: DeploymentStore.deployments.length});
+    });
```

Comment:
Yeah, it was added in https://github.com/mesosphere/marathon-ui/pull/79/files#diff-da961f43c07f401a68943c5daaec07dfR33